terraform/aws/components/lab-results/locals.tf

```diff
@@ -0,0 +1,21 @@
+locals {
+  default_tags = {
+    Project = var.project
+    Environment = var.environment
+    Component = var.component
+  }
+
+  resource_prefix = "${var.project}-${var.environment}-${var.component}"
+
+  availability_zones = ["${var.region}a", "${var.region}b", "${var.region}c"]
+
+  image_name = "${var.account_id}.dkr.ecr.${var.region}.amazonaws.com/lab-results:${var.lab-results_build_id}"
+  # Use below when the ECR repo is created by terraform in account component.
+  #image_name = "${data.terraform_remote_state.account.outputs.nhais_ecr_repo_url}:${var.nhais_build_id}"
+
+  subnet_cidrs = [
+    cidrsubnet(data.terraform_remote_state.base.outputs.nhais_cidr,2,0),
```

Comment:
Do not reuse the nhais cidr, Terraform will fail when trying to create subnet with cidr that is already assigned to different subnet in the same vpc. In the base component there is a locals_cidrs.tf file in which we specify cidrs for each componenst within environment. add a new entry there `labresults_cidr` with the next block. Then add that to outputs.tf and use the in here instead of nhais_cidr.

Reply:
subnet_cidrs reference updated in b3ab2e4d1f439e3e2a696a7bb762b0b10793427f

Reply:
New subnet for lab-results created in base component in 65b3033120d85e77cfcc52ca565aae4f79b78b55

Reply:
New output for lab-results subnet created in base component in 19d764d6f6f87186843c01b6999608a989611979